support-frontend/assets/helpers/formValidation.js

```diff
@@ -43,7 +43,7 @@ export const checkOptionalEmail: (string | null) => boolean = input => isEmpty(i
 
 export const checkGiftStartDate: (string | null) => boolean = (rawDate) => {
   const date = rawDate ? new Date(rawDate) : null;
-  return isNotEmpty(rawDate) && isNotInThePast(date) && isNotTooFarInTheFuture(date);
+  return isNotEmpty(rawDate) && isNotTooFarInTheFuture(date);
```

Comment:
Can we remove the entire `isNotInThePast` function? Is this used anywhere else in the codebase?

Reply:
It's still possible for a user to select a date in the past via the calendar component? Although unlikely. Do we know happens downstream if a user were to do this and submit the form?

Reply:
so it would send the email immediately, and it would write the provided date in to zuora/SF/data lake, which could be confusing, but is unlikely to be looked at.
It would be possible to cap it in the back end to today, if we thought it was worth it.

Reply:
Thanks @johnduffell, as long as the email is sent I think we're good here 👍 I agree the chances of someone cross-checking the provided date in to zuora/SF/data lake in this edge case scenario are low!

Reply:
Yeah, we can actually delete a couple of functions, I've just taken this and the `dateIsPast` function out.

With regards to the user selecting a date in the past, it's still not possible to do via the date picker. You can manually enter a past date in the input fields if you want, but since this won't cause an issue for the backend I don't see the point in trying to prevent it client-side. However if you enter a date in the past and hit the 'Check date' button, it will always tell you that the gift will be delivered on today's date- it didn't seem sensible to confirm to the user that yes, we would deliver their gift in 1992.

Reply:
That's great, thanks for explaining @i-hardy. I didn't know the date picker prevented historic date selection so the risk is even smaller than I'd imagined.  

> However if you enter a date in the past and hit the 'Check date' button, it will always tell you that the gift will be delivered on today's date

It sounds like from @johnduffell's comment that this is in fact the case so I don't see a problem with us telling the user that in the rare event they enter a historic date.